src/Integration.Vsix/Commands/ConnectedModeMenu/ConnectedModeMenu.vsct

```diff
@@ -13,7 +13,7 @@
         <Parent guid="ConnectedModeMenuCmdSet" id="ConnectedModeGroup"/>
         <Icon guid="guidIcons" id="bmpSonarQube"/>
         <Strings>
-          <ButtonText>Manage Server Connections...</ButtonText>
+          <ButtonText>Bind in SonarQube\SonarCloud</ButtonText>
```

Comment:
I still don't like it :/
![image](https://user-images.githubusercontent.com/60586879/206154584-97c5b4a1-779d-4646-bf08-05686727b34d.png)

Reply:
Bind to?

Reply:
Wouldnt it normally be forward slash?

Reply:
I wondered about the slash too, I couldn't find any examples